includes/definitions/discovery/liebert.yaml

```diff
@@ -60,8 +66,22 @@ modules:
                     oid: lgpPduPsTable
                     value: lgpPduPsEntryApTotal
                     num_oid: '.1.3.6.1.4.1.476.1.42.3.8.30.20.1.90.{{ $index }}'
-                    descr: 'Apparent Power for {{ $lgpPduPsEntryModel }}'
+                    descr: 'ApPwr {{ $lgpPduPsEntryModel }}'
```

Comment:
whats wrong with the old description? ApPwr I have no idea what it means

Reply:
Its too long and getting cut off in the webpage so you can't actually read the receptable name any more

Reply:
ok, looks like the current release does not have this issue any more. Will submit an unabbreviated version